src/axiom_py/query/aggregation.py

```diff
@@ -1,35 +1,11 @@
-from enum import Enum
 from dataclasses import dataclass, field as dataclass_field
 
 
-class AggregationOperation(Enum):
```

Comment:
would this break user's code if they are using it to build their queries?

Reply:
yep, breaking change. needs a bump